### PR TITLE
CI: add microbadger hooks

### PR DESCRIPTION
**Pull request by jstriebel**

### CODE REVIEW

**Comment by jstriebel** [comment]
@normanrz I did not add all the Webhooks to the CI yet, but it works for the first one (cronut). Depending on your review I will add the others in the end.

File: .circleci/config.yml [hunk 1]
```diff
@@ -8,30 +8,39 @@ jobs:
...
+              curl --silent -f -lSL https://index.docker.io/v1/repositories/scalableminds/$1/tags > /dev/null
+            }
+
             echo $DOCKER_PASS | docker login -u $DOCKER_USER --password-stdin
             for APP in $(./get_valid_order.sh); do
-              docker push scalableminds/$APP:${CIRCLE_BRANCH}__${CIRCLE_BUILD_NUM} || {
+              if does_repo_exist $APP; then
+                docker push scalableminds/$APP:${CIRCLE_BRANCH}__${CIRCLE_BUILD_NUM}
+                docker push scalableminds/$APP:${CIRCLE_BRANCH}
+                eval "MICROBADGER_HOOK=\$MICROBADGER_HOOK_$APP"
```
Comment (normanrz) [inline .circleci/config.yml]: Where does `$MICROBADGER_HOOK_$APP` come from?

Comment (jstriebel) [reply .circleci/config.yml]: this evals e.g. to `MICROBADGER_HOOK=$MICROBADGER_HOOK_cronut`
This comes from the CI env vars:
https://circleci.com/gh/scalableminds/dockerfiles/edit#env-vars

**Comment by jstriebel** [comment]
fixed the Readme on master

**Pull request merged**